honeycombio/honeycomb-opentelemetry-web
# maint: 🔧 update example app when honeycomb-opentelemetry-web files change

## Short description of the changes
Adds `dev` command to the example app. Builds the source js, starts the server, and watches `honeycomb-opentelemtry-web` files for any changes. When there's a change, re-builds the example app's js.

## How to verify that this has the expected result
- pull the branch down locally. 
```
cd examples/hello-world
npm run dev
```
- open up the example app in your browser of choice
- make a change to any of the source files in the root directory
- refresh the example app

should be running the new code
